Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java

```diff
@@ -86,20 +85,8 @@ public UserForm(LoginForm loginForm) {
      */
     @PostConstruct
     public void init() {
-        columns = new ArrayList<>();
-        try {
-            columns.add(ServiceManager.getListColumnService().getListColumnsForListAsSelectItemGroup("user"));
-            columns.add(ServiceManager.getListColumnService().getListColumnsForListAsSelectItemGroup("role"));
-            columns.add(ServiceManager.getListColumnService().getListColumnsForListAsSelectItemGroup("client"));
-            columns.add(ServiceManager.getListColumnService().getListColumnsForListAsSelectItemGroup("ldapgroup"));
-        } catch (DAOException e) {
-            Helper.setErrorMessage(e.getLocalizedMessage());
-        }
-        selectedColumns = new ArrayList<>();
-        selectedColumns.addAll(ServiceManager.getListColumnService().getSelectedListColumnsForListAndClient("user"));
-        selectedColumns.addAll(ServiceManager.getListColumnService().getSelectedListColumnsForListAndClient("role"));
-        selectedColumns.addAll(ServiceManager.getListColumnService().getSelectedListColumnsForListAndClient("client"));
-        selectedColumns.addAll(ServiceManager.getListColumnService().getSelectedListColumnsForListAndClient("ldapgroup"));
+        columns = ProjectForm.listsOfAvailableListColumns();
```

Comment:
This is wrong. Project specific columns (`project`, `template`, `workflow`, `docket`, `ruleset`) do not make sense in `UserForm`.